adm_program/system/classes/languagedata.php

```diff
@@ -73,6 +82,59 @@ public function addLanguagePath($path)
     }
 
     /**
+     * Determine the language from the browser preferences of the user.
+     * @return string Return the preferred language code of the client browser
+     */
+    private function determineBrowserLanguage($defaultLanguage)
+    {
+        if(!isset($_SERVER["HTTP_ACCEPT_LANGUAGE"]) || empty($_SERVER["HTTP_ACCEPT_LANGUAGE"]))
+        {
+            return $defaultLanguage;
+        }
+
+        $accepted = preg_split("{,\s*}", $_SERVER["HTTP_ACCEPT_LANGUAGE"]);
+        $language = $defaultLanguage;
+        $quality = 0;
+
+        if(is_array($accepted) && (count($accepted) > 0))
```

Comment:
unnecessary `()`
this is enough: `if(is_array($accepted) && count($accepted) > 0)`